fossunited/api/sidebar.py

```diff
@@ -26,7 +24,7 @@ def get_sidebar_items():
         },
     ]
 
-    if user_is_chapter_member():
+    if user_is_chapter_member(user):
```

Comment:
my lack of frappe understanding might be the cause here but i dont know how this function was working at all in the first place - this function (and others below) expects a value and it wasn't being passed a value. how did this code ever work?

Reply:
Default values are provided to be session user.

```py
def user_is_chapter_member(user: str = frappe.session.user):
    return bool(
        frappe.db.exists(
            "Has Role",
            {"role": "Chapter Team Member", "parent": user},
        )
    )


def user_is_localhost_organizer(user: str = frappe.session.user):
    return bool(
        frappe.db.exists(
            "Has Role",
            {"role": "Localhost Organizer", "parent": user},
        )
    )


def user_is_cfp_reviewer(user: str = frappe.session.user):
    return bool(
        frappe.db.exists(
            "Has Role",
            {"role": "CFP Reviewer", "parent": user},
        )
    )

```